GitCommands/Git/ExternalOperationException.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+
+namespace GitCommands
+{
+    /// <summary>
+    /// Exception class for failures of external operations,
+    /// e.g. file access or start of other executables, which always can fail due to removed or locked files.
+    /// </summary>
+    public class ExternalOperationException : Exception
```

Comment:
Could we put this class in GitCommands.Exceptions or some other non-primary namespace?